src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const marriedInfo = isMarried
+    ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
Don't nest conditions. Create variable for this value `${sex === 'm' ? 'wife' : 'husband'}`